{"name": "test_b1", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 4", "trace": "def test_b1():\n        c = 3\n        d = 4\n>       assert c == d\nE       assert 3 == 4\n\nInheri_test.py:18: AssertionError"}, "start": 1650035023286, "stop": 1650035023286, "uuid": "31c1586a-d51a-44be-891d-072e176c254e", "historyId": "eb2edf6766126991e940a1ced3633225", "testCaseId": "33d4217c88624651dadb091335cab84a", "fullName": "Inheri_test#test_b1", "labels": [{"name": "suite", "value": "Inheri_test"}, {"name": "host", "value": "MSDT120"}, {"name": "thread", "value": "19292-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Inheri_test"}]}